Add multiturn documentation (#190)
pylhc/pylhc.github.io: docs/guis/multiturn/scheduler.md

# AC-Dipole Scheduler

> Once all the steps have been followed in the [Multiturn GUI](gui.md), you can start the measurements with the AC-dipole Scheduler.

Reviewer: Maybe we don't make it read like it's a mandatory step.

```suggestion
It is possible to schedule a kick plan ahead of time with the Scheduler, before triggering its execution.
```

The multiturn application provides a scheduler to run predefined kick amplitudes one after another.
This is useful to automate the process of running measurements with different kick amplitudes, which can speed up the process of collecting data for analysis, and potentially reduces the number of people required on shift.

> The scheduler can be found in the `ACDipoleScheduled` tab of the Multiturn GUI, see the screenshot below:

Reviewer: ```suggestion
The scheduler can be found in the `ACDipoleScheduled` tab of the Multiturn GUI, as illustrated below:
```

<figure>
    <center>
    <img src="../../assets/images/multiturn_gui/prewritten_schedules.png" width="85%" alt="Prewritten Kick Schedules" />
    <figcaption>Prewritten Kick Schedules</figcaption>
    </center>
</figure>

# Running a Kick Schedule
Once you have created or loaded a kick schedule, it is necessary to setup the horizontal and vertical kick tune deltas. Refer to the [AC-Dipole Measurements](acdipole.md) page for context on what to set these to. 

Now all the settings are ready, it is possible to run the kick schedule from the very beginning of the table by selecting the button `Acquire with ACDipoleScheduled excitation` at the bottom left of the GUI (highlighted in yellow). This will start the measurement with the first row of the table, and will automatically move to the next row once the measurement is complete. 

> [!INFO] If you are running a kick schedule and you would like to cancel the acquisition, you can do so by selecting the button `Stop`. But if you have just done a kick on one amplitude, pressing `Stop` will not stop the next kick from being executed, instead it will stop after the next kick is complete. There is no way to cancel the next kick once it is waiting for the AC-Dipole to be ready. 

Reviewer: Different admonition style for mkdocs-material than Github 😅 

```suggestion
!!! info
    To cancel the acquisition while running a kick schedule, press the `Stop` button. Note that if a kick was just done when pressing `Stop`, the schedule will stop after the next kick is complete. There is no way to cancel the next kick once it is waiting for the AC-Dipole to be ready. 
```
